src/Runner.Common/JobServerQueue.cs

```diff
@@ -275,7 +275,7 @@ private async Task ProcessWebConsoleLinesQueueAsync(bool runOnce = false)
                         {
                             Trace.Info($"Skip {batchedLines.Count - 2} batches web console lines for last run");
                             batchedLines = batchedLines.TakeLast(2).ToList();
-                            batchedLines[0].Insert(0, "...");
+                            batchedLines[0].Insert(0, new TimelineRecordLogLine("...", 0));
```

Comment:
should the line number be `0` or whatever the current number is?

Reply:
What does 0 mean to the server?

Reply:
We can remove this. It's removed in ADO too.